src/Form/SettingsForm.php

```diff
@@ -196,6 +146,53 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       );
     }
 
+    if(count($validate_plugins_fallbacks)){
+      $form['tfa_fallback'] = array(
+        '#type' => 'fieldset',
+        '#title' => t('Validation fallback plugins'),
+        '#description' => t('Fallback plugins and order. Note, if a fallback plugin is not setup for an account it will not be active in the TFA form.'),
+        '#states' => $enabled_state,
+        '#tree' => TRUE,
+      );
+
+      $enabled_fallback_plugins = \Drupal::config('tfa.settings')->get('fallback_plugins');
+      foreach ($validate_plugins_fallbacks as $plugin => $fallbacks) {
+        $fallback_state  = array(
+          'visible' => array(
+            ':input[name="tfa_validate"]' => array('value' => $plugin)
+          )
+        );
+        if(count($fallbacks)) {
+          foreach ($fallbacks as $fallback) {
+            $order = (@$enabled_fallback_plugins[$plugin][$fallback]['weight']) ?: -2;
+            $fallback_value = (@$enabled_fallback_plugins[$plugin][$fallback]['enable']) ?: 1;
+            $form['tfa_fallback'][$plugin][$fallback] = array(
+              'enable' => array(
+                '#title'         => $validate_plugins_labels[$fallback],
```

Comment:
The fallbacks are basically validation plugins.
I pull in the labels of all validation plugins in that variable so used that same one here.